community/libtorrent/sources

```diff
@@ -0,0 +1 @@
+http://rtorrent.net/downloads/libtorrent-0.13.8.tar.gz
```

Comment:
No https source available?

Reply:
Have changed to use https.  For some reason they use a http link on their download page.